ops/deploy-ccs.groovy

```diff
@@ -118,32 +90,61 @@ def buildAppAmis(String gitBranchName, String gitCommitId, AmiIds amiIds, AppBui
 	dir('ops/ansible/playbooks-ccs'){
 
 		writeJSON file: "${workspace}/ops/ansible/playbooks-ccs/extra_vars.json", json: amis
+			packerBuildAmis(amiIds.platinumAmiId, gitBranchName, gitCommitId,
+					"../../packer/build_bfd-all.json")
+
+			amiIdsWrapper.platinumAmiId = amiIds.platinumAmiId
+			amiIdsWrapper.bfdPipelineAmiId = extractAmiIdFromPackerManifest(readFile(
+						file: "${workspace}/ops/ansible/playbooks-ccs/manifest_data-pipeline.json"))
+			amiIdsWrapper.bfdServerAmiId = extractAmiIdFromPackerManifest(readFile(
+						file: "${workspace}/ops/ansible/playbooks-ccs/manifest_data-server.json"))
+			amiIdsWrapper.bfdMigratorAmiId = extractAmiIdFromPackerManifest(readFile(
+						file: "${workspace}/ops/ansible/playbooks-ccs/manifest_db-migrator.json"))
+			amiIdsWrapper.bfdServerLoadAmiId = extractAmiIdFromPackerManifest(readFile(
+						file: "${workspace}/ops/ansible/playbooks-ccs/manifest_server-load.json"))
+	}
+
+	return amiIdsWrapper
+}
+
+/**
+ * Builds the Docker Host AMI.
+ *
+ * @param gitBranchName the name of the Git branch this build is for
+ * @param gitCommitId the hash/ID of the Git commit that this build is for
+ * @param the ID of the base "platinum" AMI to use for new instances/AMIs
+ * @return the AMI id of the Docker Host AMI
+ * @throws RuntimeException An exception will be bubbled up if the AMI-builder tooling returns a non-zero exit code.
+ */
+def buildDockerHostAmi(String gitBranchName, String gitCommitId, String platinumAmiId) {
+	dir('ops/packer'){
+		packerBuildAmis(platinumAmiId, gitBranchName, gitCommitId, "./build_bfd-docker-host.json")
```

Comment:
We didn't discuss this at length, but I really appreciate you decoupling this from the monolithic packer manifest and encoding it into a distinct, docker-host manifest. 👍